assets/app/scripts/controllers/otherResources.js

```diff
@@ -74,7 +74,9 @@ angular.module('openshiftConsole')
       DataService.list({
```

Comment:
The ` ProjectsService.get($routeParams.project)` is nice and terse here!  Only suggestion would be to not put anything on `$scope` that isn't for view (the `context`).  Since `loadKind()` isn't nested in the `.then()`, we could just `var context = {}` at the top, replace it when the project request returns, and pass `DataService.list({}, context, null, {})`.